core/idl/wallet/ripple/ripple_like_wallet.djinni

```diff
@@ -38,6 +38,8 @@ RippleLikeTransaction = interface +c {
     getMemos(): list<RippleLikeMemo>;
     # Add a memo to a transaction.
     addMemo(memo: RippleLikeMemo);
+    # An arbitrary unsigned 32-bit integer that identifies a reason for payment or a non-Ripple account
```

Comment:
s/or/for

Reply:
Also, you only need 32 bit.

Reply:
I took the line from Ripple's doc :trollface: 
For 32-bit remark, we don't have unsigned 32-bit in Djinni so ...